src/Service/ProgrammesAggregationService.php

```diff
@@ -30,6 +30,14 @@ public function __construct(
         parent::__construct($repository, $mapper, $cache);
     }
 
+    public function countStreamableOnDemandEpisodes(
+        Programme $programme,
+        $ttl = CacheInterface::NORMAL,
+        $nullTtl = CacheInterface::SHORT
+    ): int {
+        return $this->countStreamableDescendantsByType($programme, 'Episode', $ttl, $nullTtl);
```

Comment:
Why the split here?

Reply:
To be consistent with `findStreamableDescendantsByType()`